src/styles/index.scss

```diff
@@ -1,3 +1,129 @@
+$bc-image: url(/src/images/star-active.svg);
+$main-acent: #060b35;
+$secondary: #616070;
+$btn-white-color: #ffffff;
+$btn-turquoise-color: #00acdc;
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $main-acent;
+    margin-bottom: 4px;
+    margin-top: 0;
+  }
+
+
+  &__code {
+    line-height: 14px;
+    text-align: left;
+    margin-bottom: 16px;
+    margin-top: 0;
+    color: $secondary;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+    margin-top: 0;
+    color: $secondary;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+    color: #060b35;
+  }
+
+  &__price-header {
+    font-size: 12px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    margin-top: 0;
+    height: 16px;
+    width: 166px;
+  }
+
+  &__review-count {
+    text-align: right;
+    color: $main-acent;
+  }
+
+  &__btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: $btn-white-color;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    width: 100%;
+    height: 40px;
+    box-sizing: border-box;
+    border: 1px solid $btn-turquoise-color;
+    border-radius: 5px;
+    background: $btn-turquoise-color;
+
+    &:hover {
+      background: #fff;
+      color: $btn-turquoise-color;
+    }
+  }
+}
+
+.stars {
```

Comment:
<img width="610" alt="image" src="https://github.com/user-attachments/assets/261f2817-6859-4f5e-868c-b30af0a5fd60">
each BEM block should be in a separate file